features/step_definitions/neg_search_steps.rb

```diff
@@ -0,0 +1,7 @@
+Then('I should see a dropdown with no results message') do
+  expect(@homepage).to have_text(Constants::NO_RESULTS_MESSAGE)
+end
+
+Then('I should see the no results message') do
+  expect(@homepage.empty_results_text).to be_visible
+end
```

Comment:
i think we can combine this verification into one(if in the 2nd verification we are checking the same text that is in the constants), so it will look like `empty results text` is equal to `no results message`

Reply:
The `no_results_ message` and `empty_results_text` are different messages with different texts. The `no_results_ message` is a message from the dropdown when we type into the search field and do not click the search button. And `empty_results_text` is a text that appears when we click the search button.